src/pages/iou/SplitBillDetailsPage.js

```diff
@@ -34,6 +36,12 @@ const propTypes = {
     /** Array of report actions for this report */
     reportActions: PropTypes.shape(reportActionPropTypes),
 
+    /** The current transaction */
+    transaction: PropTypes.shape(transactionPropTypes).isRequired,
```

Comment:
Getting prop warning that the transaction is undefined
<img width="564" alt="Screenshot 2023-10-11 at 8 23 56 PM" src="https://github.com/Expensify/App/assets/16493223/4dfca0bf-3d91-4b81-b6fd-1211df017662">